source/includes/_users.md

```diff
@@ -0,0 +1,302 @@
+# Users
+
+## The User Resource
+
+<aside class="notice">
+This section describes API features that are part of the Cloudify premium edition
+</aside>
+
+> `Note`
+
+```python
+# include this code when using cloudify python client-
+from cloudify_rest_client import CloudifyClient
+client = CloudifyClient(
+        host='<manager-ip>',
+        username='<manager-username>',
+        password='<manager-password>',
+        tenant='<manager-tenant>')
+```
+
+```html
+
+```
+
+Since Cloudify 4.0 there is users Cludify users management.
```

Comment:
Since Cloudify 4.0 **, Cloudify user management has been added** ~~there is users Cludify users management.~~